src/compiler/emitter.ts

```diff
@@ -45,7 +45,7 @@ var __assign = (this && this.__assign) || Object.assign || function(t) {
         const restHelper = `
 var __rest = (this && this.__rest) || function (s, e) {
     var t = {};
-    for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p) && !e.indexOf(p))
+    for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p) && e.indexOf(p) === -1)

```

Comment:
The test is for *not* found. But I guess i could use `< 0`.

@timsuchanek Typescript's emit tries to balance readability, speed and brevity. Brevity is the least important of the three, though, and I think `!~e.indexOf(p)` is less readable than `e.indexOf(p) < 0`.